packages/app/src/lib/sanity.tsx

```diff
@@ -49,7 +48,7 @@ export const client = sanityClient(config);
  **/
 // export const urlFor = (source: SanityImageSource) => builder.image(source);
 
-export function localize<T>(value: T | T[], languages: LanguageKey[]): T {
+export function localize<T>(value: T | T[], languages: any[]): T {
```

Comment:
```suggestion
export function localize<T>(value: T | T[], languages: string[]): T {
```
Then cast it where the languages is used. This way you keep the public interface kind of strict